### PR TITLE
Update: Swift.Toolchian.5.7.3

### PR DESCRIPTION
**Pull request by compnerd**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94440)

### CODE REVIEW

File: manifests/s/Swift/Toolchain/5.7.3/Swift.Toolchain.installer.yaml [hunk 1]
```diff
@@ -21,10 +21,7 @@ Dependencies:
   - PackageIdentifier: Git.Git
 Installers:
 - Architecture: x64
-  InstallerUrl: https://download.swift.org/swift-5.7.2-release/windows10/swift-5.7.2-RELEASE/swift-5.7.2-RELEASE-windows10.exe
-  InstallerSha256: F523F8D364135738D9E7419F987A654A15159424434A13FC480079BCFCE78C75
-  AppsAndFeaturesEntries:
-    - DisplayVersion: 5.7.0
-      DisplayName: Swift Developer Package for Windows x86_64
+  InstallerUrl: https://download.swift.org/swift-5.7.3-release/windows10/swift-5.7.3-RELEASE/swift-5.7.3-RELEASE-windows10.exe
+  InstallerSha256: 179b44cf7f1ce5c74dc864207119464ff41526a2cc2257cfd2ccef65bc25b231
```
Comment (mdanish-kh) [inline manifests/s/Swift/Toolchain/5.7.3/Swift.Toolchain.installer.yaml:25]:
```suggestion
  InstallerSha256: 179b44cf7f1ce5c74dc864207119464ff41526a2cc2257cfd2ccef65bc25b231
  ProductCode: '{71166b25-5db3-454d-b4d8-cf96b0caf289}'
  AppsAndFeaturesEntries:
    - DisplayName: Swift Developer Package for Windows x86_64
```
We can keep DisplayName as it helps WinGet for better upgrade matching. Suggested ProductCode for the same reason.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]
Note: The pipelines are down at the moment (https://github.com/microsoft/winget-pkgs/pull/94440#issuecomment-1400621568), so this PR may take some time to get merged.

**Comment by compnerd** [comment]
@mdanish-kh thanks for the heads up!

**Pull request merged**